l10n_it_fatturapa_out_rc/wizard/wizard_export_e_invoice.py

```diff
@@ -0,0 +1,127 @@
+from odoo import models, _
+from odoo.exceptions import UserError
+from odoo.addons.l10n_it_account.tools.account_tools import encode_for_export
+from odoo.addons.l10n_it_fatturapa.bindings.fatturapa import (
+    IdFiscaleType,
+    AnagraficaType,
+    IndirizzoType
+)
+
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    def exportInvoiceXML(
+        self, company, partner, invoice_ids, attach=False, context=None
+    ):
+        if context is None:
+            context = {}
+        invoices = self.env["account.invoice"].browse(invoice_ids)
+        invoices_with_rc = False
+        invoices_without_rc = False
+        for invoice in invoices:
+            if invoice.rc_purchase_invoice_id:
+                invoices_with_rc = True
+            else:
+                invoices_without_rc = True
+        if invoices_with_rc and invoices_without_rc:
+            raise UserError(_(
+                "Selected invoices are both with and without reverse charge. You "
+                "should selected a smaller set of invoices"))
+        rc_suppliers = invoices.mapped("rc_purchase_invoice_id.partner_id")
```

Comment:
In caso di tipo rc `supplier` l'`rc_supplier`/cedente è il partner dell'autofattura emessa, mentre il cessionario è la `company`.

Reply:
Non ti seguo: nel caso di comunicazione a SDI il tipo `supplier` non va proprio usato

Reply:
Non avevo letto l'issue, ok. Forse è il caso di segnalare che l'opzione `supplier` non è valida per l'uso con SDI.

Reply:
:+1: aggiunto `CONFIGURE.rst`